sdks/web-sdk/src/declarations.d.ts

```diff
@@ -0,0 +1,28 @@
+// Should use Posthog's types here instead
+import { AnyRecord } from './types';
+import { DevMocks } from './lib/contexts/app-state';
+
+interface IPosthog {
```

Comment:
We can remove all references to posthog

Reply:
Removed src/analytics.ts as its currently the only file using posthost. 
![image](https://user-images.githubusercontent.com/61207713/199472925-8b500fb7-d415-40ff-a820-3a74e9a68968.png)